### PR TITLE
Remove FIRs without boundary definition

### PR DESCRIPTION
**Pull request by kusterjs**
Consistency checked and FIRs removed without a boundary, they won't light up, so their definition is of no use

### CODE REVIEW

File: VATSpy.dat [hunk 1]
```diff
@@ -18270,9 +18253,6 @@ VVVV|Hanoi||VVVV
 VVTS|Ho-Chi-Minh||VVTS
 VYYF|Rangoon|VYYY|VYYF
 WAAF|Ujung Pandang||WAAF
-WAAF-B|Ujung Pandang (Bali)|WAAF_B|WAAF_B
```
Comment (marvk) [inline VATSpy.dat:18273]: WAAF-B, WAAF-W and WAAF-E exist as FIR-Boundaries

https://github.com/vatsimnetwork/vatspy-data-project/blob/f840d74eee3032cc77da5116dbd4563aad6009e6/FIRBoundaries.dat#L20443

https://github.com/vatsimnetwork/vatspy-data-project/blob/f840d74eee3032cc77da5116dbd4563aad6009e6/FIRBoundaries.dat#L20451

https://github.com/vatsimnetwork/vatspy-data-project/blob/f840d74eee3032cc77da5116dbd4563aad6009e6/FIRBoundaries.dat#L20507

**Review by kusterjs — COMMENTED** [review]
WAAF sectors added again

**Comment by marvk** [comment]
@kusterjs There's more FIRs using an underscore instead of a dash than WAAF. I've referenced this in my third point in #197

**Comment by kusterjs** [comment]
yeah, well but one of the two should reference to the boundary name, shouldn't it?

**Comment by marvk** [comment]
I'm not saying it's a good thing, just saying that this isn't a sole issue for WAAF :-) i.e. the following:

https://github.com/vatsimnetwork/vatspy-data-project/blob/fd3fc9673f64b858ea66f3348ce39d1c663b97b7/VATSpy.dat#L17769

https://github.com/vatsimnetwork/vatspy-data-project/blob/fd3fc9673f64b858ea66f3348ce39d1c663b97b7/VATSpy.dat#L17828

https://github.com/vatsimnetwork/vatspy-data-project/blob/fd3fc9673f64b858ea66f3348ce39d1c663b97b7/VATSpy.dat#L18018

All I'm saying is that I'm not sure if all of these are intended either. In any case, it's probably slightly off topic for this review, just wanted to let you know.

**Comment by prryplatypus** [comment]
While it's true that these FIRs don't have an FIR definition, it is possible that they are referenced from the airports section in `VATSPY.dat`. I would like to leave all FIRs that have airports which belong to them, but I'm happy to remove those that don't. Mind checking this for me, please? 🙂

**Comment by marvk** [comment]
@prryplatypus what exactly do you need checked? FIR definitions that don't have their ICAO match either an Airport, a FIRB ICAO or a UIR?

**Comment by prryplatypus** [comment]
I want to make sure that we leave the FIR definitions for those FIRs that are referenced at least once from the `[Airports]` section. For example, the `RJJJ` FIR is referenced by 11 different airports (for example [`RORE`](https://github.com/vatsimnetwork/vatspy-data-project/blob/5ea4314d94b3b6cf677110bd03c143aec0860200/VATSpy.dat#L10694)), so I would like to leave at least one of the FIR definitions for `RJJJ`, even if there is no boundary for it in the `FIRBoundaries.dat` file.

Ideally someone could submit a PR to create its boundaries instead so we can kill two birds with one stone 🐦

**Comment by marvk** [comment]
I got you. The only FIRs that I could find where neither the FIR ICAO, the FIR PrefixPosition nor the unnamed fourth FIR parameter is referencing either an Airport or a FIRB are:

```
VatsimFlightInformationRegion(icao=GMMM_E, name=Casablanca, prefixPosition=, unknown1=GMMM_E)
VatsimFlightInformationRegion(icao=GMMM_N, name=Casablanca, prefixPosition=, unknown1=GMMM_N)
VatsimFlightInformationRegion(icao=GMMM_S, name=Casablanca, prefixPosition=, unknown1=GMMM_S)
VatsimFlightInformationRegion(icao=URFV, name=Simferopol, prefixPosition=URFF, unknown1=)
```

When looking *only* for FIR ICAOs, the following FIRs are filtered:

```
VatsimFlightInformationRegion(icao=EBUR, name=Brussels Upper, prefixPosition=, unknown1=EBBU)
VatsimFlightInformationRegion(icao=EGG1, name=Shanwick, prefixPosition=SC, unknown1=EGGX)
VatsimFlightInformationRegion(icao=EGG2, name=Shanwick Clearance Delivery, prefixPosition=SC-D, unknown1=EGGX)
VatsimFlightInformationRegion(icao=EGG3, name=Shanwick Oceanic Routing Clearance Authority, prefixPosition=S-ORCA, unknown1=EGGX)
VatsimFlightInformationRegion(icao=EGG3, name=Shanwick Oceanic Routing Clearance Authority, prefixPosition=SORCA, unknown1=EGGX)
VatsimFlightInformationRegion(icao=EGG4, name=Shanwick Oceanic Clearance Delivery, prefixPosition=EGGX-D, unknown1=EGGX)
VatsimFlightInformationRegion(icao=EGG4, name=Shanwick Oceanic Clearance Delivery, prefixPosition=EGX-D, unknown1=EGGX)
VatsimFlightInformationRegion(icao=EGG4, name=Shanwick Oceanic Clearance Delivery, prefixPosition=EG-D, unknown1=EGGX)
VatsimFlightInformationRegion(icao=EGG4, name=Shanwick Oceanic Clearance Delivery, prefixPosition=GX-D, unknown1=EGGX)
VatsimFlightInformationRegion(icao=EGG4, name=Shanwick Oceanic Clearance Delivery, prefixPosition=SH-D, unknown1=EGGX)
VatsimFlightInformationRegion(icao=EGG4, name=Shanwick Oceanic Clearance Delivery, prefixPosition=SR-D, unknown1=EGGX)
VatsimFlightInformationRegion(icao=EGG4, name=Shanwick Oceanic Clearance Delivery, prefixPosition=SHW-D, unknown1=EGGX)
VatsimFlightInformationRegion(icao=EGG4, name=Shanwick Oceanic Clearance Delivery, prefixPosition=SHAN-D, unknown1=EGGX)
VatsimFlightInformationRegion(icao=EGOM, name=Spadeadam Range, prefixPosition=SPAD, unknown1=EGTT)
VatsimFlightInformationRegion(icao=EGQN, name=Buchan Air Defence, prefixPosition=BUCH, unknown1=EGPX)
VatsimFlightInformationRegion(icao=EGQN, name=Buchan Air Defence, prefixPosition=BUAD, unknown1=EGPX)
VatsimFlightInformationRegion(icao=EGUJ, name=Neatishead Air Defence, prefixPosition=NEAT, unknown1=EGTT)
VatsimFlightInformationRegion(icao=EHML, name=Bandbox CRC (Military), prefixPosition=, unknown1=EHAA)
VatsimFlightInformationRegion(icao=ESSR, name=Stockholm RTC - Sweden, prefixPosition=, unknown1=ESOS)
VatsimFlightInformationRegion(icao=GMMM_E, name=Casablanca, prefixPosition=, unknown1=GMMM_E)
VatsimFlightInformationRegion(icao=GMMM_N, name=Casablanca, prefixPosition=, unknown1=GMMM_N)
VatsimFlightInformationRegion(icao=GMMM_S, name=Casablanca, prefixPosition=, unknown1=GMMM_S)
VatsimFlightInformationRegion(icao=HLLB, name=Benina, prefixPosition=BEN, unknown1=HLLL)
VatsimFlightInformationRegion(icao=KZA1, name=San Francisco Oceanic, prefixPosition=SF, unknown1=KZAK)
VatsimFlightInformationRegion(icao=KZA1, name=San Francisco Oceanic, prefixPosition=SFR, unknown1=KZAK)
VatsimFlightInformationRegion(icao=KZA1, name=San Francisco Oceanic, prefixPosition=SFRO, unknown1=KZAK)
VatsimFlightInformationRegion(icao=LFUM, name=Marseille Upper, prefixPosition=, unknown1=LFMM)
VatsimFlightInformationRegion(icao=LLGV, name=Hagav, prefixPosition=HAGAV, unknown1=LLLL)
VatsimFlightInformationRegion(icao=LLPL, name=Pluto, prefixPosition=PLUTO, unknown1=LLLL)
VatsimFlightInformationRegion(icao=LPAM, name=Lisboa Military, prefixPosition=, unknown1=LPPC)
VatsimFlightInformationRegion(icao=NSTU, name=Pago Pago, prefixPosition=, unknown1=NZZO)
VatsimFlightInformationRegion(icao=SPI1, name=Lima North, prefixPosition=SP-N, unknown1=SPIM)
VatsimFlightInformationRegion(icao=SPI2, name=Lima South, prefixPosition=SP-S, unknown1=SPIM)
VatsimFlightInformationRegion(icao=SPI3, name=Lima North East, prefixPosition=SP-E, unknown1=SPIM)
VatsimFlightInformationRegion(icao=SPI3, name=Lima North East, prefixPosition=SP-NE, unknown1=SPIM)
VatsimFlightInformationRegion(icao=URFV, name=Simferopol, prefixPosition=URFF, unknown1=)
```

**Comment by prryplatypus** [comment]
@kusterjs Could you have a look at what's been commented above for next AIRAC cycle?

**Comment by kusterjs** [comment]
@prryplatypus all resolved

**Review by Adrian2k — APPROVED** [review]

**Review by NelisV — APPROVED** [review]

**Pull request merged**